src/Microsoft.AspNet.Mvc.Razor/RazorPage.cs

```diff
@@ -157,7 +159,7 @@ public TTagHelper CreateTagHelper<TTagHelper>() where TTagHelper : ITagHelper, n
         /// </remarks>
         public void StartWritingScope()
         {
-            StartWritingScope(new StringWriter());
+            StartWritingScope(new StringCollectionTextWriter(Encoding.UTF8));
```

Comment:
I'm not aware of a system-wide setting.  but in normal use `RazorTextWriter` is passed `ViewContext.Writer.Encoding` when created.  AFAICT that value is always `Encodings.UTF8EncodingWithoutBOM`.